css/styles.css

```diff
@@ -0,0 +1,108 @@
+header {
+  background-color: #62605c;
+  font-family: 'Anton', sans-serif;
+  color: white;
+  display: flex;
+  justify-content: flex-end;
+  height: 80px;
+}
+
+.top-links {
+  display: flex;
+  justify-content: space-evenly;
+  list-style: none;
+  margin-top: 0;
+  width: 50%;
+}
+
+.container {
+  background-color: #ffa500;
+  display: flex;
+  flex-direction: column;
+  height: 800px;
+}
+
+h2 {
+  color: white;
+  font-family: 'Anton', sans-serif;
+  font-size: 80px;
+  margin: 0;
+}
+
+.main-one {
+  text-align: center;
+  height: 266px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.main-two {
+  height: 532px;
+}
```

Comment:
main-one and main-two? What are these for? Try to use names that describe your purpose. You should be using a grid system, I recommend grabbing the `grid.css` file from your solution for `css-grid-classes`, and use that grid.